src/posthog-core.js

```diff
@@ -1494,7 +1494,11 @@ PostHogLib.prototype.sentry_integration = function (_posthog, organization, proj
         addGlobalEventProcessor((event) => {
             if (event.level !== 'error' || !_posthog.__loaded) return event
             if (!event.tags) event.tags = {}
-            event.tags['PostHog URL'] = _posthog.config.api_host + '/person/' + _posthog.get_distinct_id()
+            event.tags['PostHog Person URL'] = _posthog.config.api_host + '/person/' + _posthog.get_distinct_id()
+            if (_posthog.sessionRecordingStarted()) {
+                event.tags['PostHog Recording URL'] =
+                    _posthog.config.api_host + '/recordings/#sessionRecordingId=' + _posthog.sessionManager.sessionId
```

Comment:
What's the difference here between accessing the sessionId directly, vs, calling `getSessionAndWindowId` ?

.. And where should you use what? Since nowhere else tries to directly access instance variables for the sessionManager, would be nice to keep that pattern going, using only accessors to get these values, as I see other logic happening on these accessors.

Reply:
This is a realllly good question. And it's way too confusing right now. 

The logic in the `getSessionAndWindowId` is around checking to see if we should cycle the `session_id` or not (e.g. has it been >30 minutes since the last qualifying event). Where a qualifying event is a "user interaction" and not some random DOM manipulation event. The issue is that the logic in determining if something is a "qualifying event" is kind of coupled with recordings right now - meaning only recording events can be disqualified.

In the sentry case here, we don't want to cycle the session_id (because then, the recording linked to in sentry wouldn't be the one shown in the link), so to avoid the "potential cycle", I just accessed the property directly.

This should really be refactored, so it's easier to understand + work with. When I first wrote it, session_ids we're only a concept in recordings, and it's growing beyond that now.

Torn on whether I should block merging this on refactoring, so I definitely do it 😛. Going to make the call Monday